### ISSUE TITLE
Deactivate Drupal\node\Node

### ISSUE THREAD

**Issue by Peekmo** [makinacorpus/drupal-sf-dic#1]
Hey !

First of all, thank you for this module.
I have an issue with the module `crumbs` : `Recoverable fatal error: Argument 1 passed to crumbs_MultiPlugin_EntityFindSomething::entityFind() must be an instance of stdClass, instance of Drupal\node\Node given`

Is there a way to not return Drupal\node\Node and just return the default stdClass ?

Thanks,
Peekmo

**Comment by pounard** [comment]
I never really though of that, I did experience the bug with pathauto but I finally patched it, then later disabled it in flavor of custom code.

Even without patching sf_dic module, it can be manually done overriding the controller class in the hook_entity_info_alter() and it should do the trick.

I'm tagging this issue as enhancement, I should provide a few variable to disable this controller override.
